test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -75,12 +77,60 @@ var testData = testDataType{
 	workspace2Disambiguited: &tenancyv1beta1.Workspace{ObjectMeta: metav1.ObjectMeta{Name: "workspace2--1"}},
 }
 
+func createOrgMemberRoleForGroup(ctx context.Context, kubeClient kubernetes.Interface, orgClusterName string, groupNames ...string) error {
+	_, orgName, err := helper.ParseLogicalClusterName(orgClusterName)
+	if err != nil {
+		return err
+	}
+
+	if _, err := kubeClient.RbacV1().ClusterRoles().Create(ctx, &rbacv1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: orgName + "-member",
```

Comment:
can be follow-up for the controller work: this should be `org-<orgName>-member` or so

Reply:
fixed in commit 56947bcb00c6b1746f053060eb32042705a133a2